checkipa

```diff
@@ -255,7 +275,10 @@ class IntegrityCheck(object):
                 txt.append('  %s' % (udid))
         else:
             txt.append('  Provisioned Devices (0):')
-
+        txt.append('  Package size: %2.2f GB' % (self.ipa_size/1000000000.0))
```

Comment:
Don, I think you mean GibiByte:
https://en.wikipedia.org/wiki/Gibibyte 
Apple spec says GB not GiB, and given the ambiguity, I would rather be more restrictive.